calico-enterprise/getting-started/compatibility.mdx

```diff
@@ -54,7 +54,7 @@ Kubernetes version support aligns with [upstream Kubernetes](#kubernetes-kubeadm
 
 | {{prodname}} version | kOps and Kubernetes versions | {{prodname}} support                                         |
 | -------------------- | ---------------------------- | ------------------------------------------------------------ |
-| 3.19                 | 1.26 - 1.28                  | - {{prodname}} CNI with network policy<br />- AWS CNI with {{prodname}} network policy |
+| 3.19                 | 1.27 - 1.29                  | - {{prodname}} CNI with network policy<br />- AWS CNI with {{prodname}} network policy |
```

Comment:
Since the upstream support is between k8s 1.28 - 1.30 other platforms' support should be based on similar version.

There are 2 outliers here though MKE and TKG, since the max they support is 1.27.

```suggestion
| 3.19                 | 1.28 - 1.29                  | - {{prodname}} CNI with network policy<br />- AWS CNI with {{prodname}} network policy |
```